contracts/accountants/HealthCheckAccountant.sol

```diff
@@ -66,8 +75,29 @@ contract HealthCheckAccountant {
         _;
     }
 
+    modifier onlyVaultManagers() {
+        _checkVaultManagers();
+        _;
+    }
+
+    modifier onlyAddedVaults() {
+        _checkVaultIsAdded();
+        _;
+    }
+
     function _checkFeeManager() internal view virtual {
-        require(feeManager == msg.sender, "!fee manager");
+        require(msg.sender == feeManager, "!fee manager");
+    }
+
+    function _checkVaultManagers() internal view virtual {
+        require(
+            msg.sender == feeManager || msg.sender == vaultManager,
+            "!vault manager"
+        );
+    }
+
+    function _checkVaultIsAdded() internal view virtual {
+        require(vaults[msg.sender], "!authorized");
```

Comment:
small: i would suggest slightly more informative message like "vault not added" rather than generic access control.

Reply:
https://github.com/yearn/vault-periphery/pull/27/commits/b0aa4889148b14212c0914a3276e430c345b22ef